internal/pkg/cli/env_delete_test.go

```diff
@@ -222,14 +225,55 @@ func TestDeleteEnvOpts_Execute(t *testing.T) {
 				}
 			},
 
-			wantedError: errors.New("service 'frontend, backend' still exist within the environment test"),
+			wantedError: errors.New(`service "frontend, backend" still exist within the environment test`),
+		},
+		"returns error when more pipelines are using the env": {
+			given: func(t *testing.T, ctrl *gomock.Controller) *deleteEnvOpts {
+				rg := mocks.NewMockresourceGetter(ctrl)
+				rg.EXPECT().GetResources(gomock.Any()).Return(&resourcegroupstaggingapi.GetResourcesOutput{
+					ResourceTagMappingList: []*resourcegroupstaggingapi.ResourceTagMapping{},
+				}, nil)
+
+				lister := mocks.NewMockdeployedPipelineLister(ctrl)
+				lister.EXPECT().ListDeployedPipelines("phonetool").Return([]deploy.Pipeline{
+					{
+						ResourceName: "mockResourceName",
+						Name:         "mockName",
+					},
+				}, nil)
+
+				getter := mocks.NewMockpipelineGetter(ctrl)
+				getter.EXPECT().GetPipeline("mockResourceName").Return(&codepipeline.Pipeline{
+					Stages: []*codepipeline.Stage{
+						{
+							Name: "DeployTo-test",
```

Comment:
This test having the name hardcoded helps with the durability too because it should catch any breaking name change 👍